WordPressKit/RemoteSiteDesign.swift

```diff
@@ -22,5 +24,6 @@ public struct RemoteSiteDesign: Codable {
         demoURL = try map.decode(String.self, forKey: .demoURL)
         screenshot = try? map.decode(String.self, forKey: .screenshot)
         themeSlug = try? map.decode(String.self, forKey: .themeSlug)
+        isBlog = (try? map.decode(Bool.self, forKey: .isBlog)) ?? false
```

Comment:
I'm not 100% sure about how we'll make this distinction but went ahead and added one path here. Worst-case scenario, `isBlog` can become a computed property based on whatever values are supplied by the service.